truss/tests/test_model_inference.py

```diff
@@ -137,6 +137,58 @@ def _test_invocations(expected_code):
         assert not _test_invocations(expected_code=200)
 
 
+@pytest.mark.integration
+def test_streaming_truss():
+    with ensure_kill_all():
+        truss_root = Path(__file__).parent.parent.parent.resolve() / "truss"
+        truss_dir = truss_root / "test_data" / "test_streaming_truss"
+        tr = TrussHandle(truss_dir)
+
+        _ = tr.docker_run(local_port=8090, detach=True, wait_for_server_ready=False)
```

Comment:
you can just set `wait_for_server_ready=True` or drop it from kwargs, then just jjump to the tests at 171. Adding the liveness/readiness probes here is not related to streaming and is test elsewhere.